README.md

```diff
@@ -1,13 +1,48 @@
 
-## BETA ACKNOWLEDGEMENT
+# Adobe Experience Platform Assurance SDK
 
-AEP Assurance Android is currently in Beta. Use of this code is by invitation only and not otherwise supported by Adobe. Please contact your Adobe Customer Success Manager to learn more.
+## About this project
 
-By using the Beta, you hereby acknowledge that the Beta is provided "as is" without warranty of any kind. Adobe shall have no obligation to maintain, correct, update, change, modify or otherwise support the Beta. You are advised to use caution and not to rely in any way on the correct functioning or performance of such Beta and/or accompanying materials.
+AEP Assurance Android SDK is a mobile extension for Adobe Experience Platform that allows integrating with [Adobe Experience Platform Assurance](https://developer.adobe.com/client-sdks/documentation/platform-assurance/) to help 
+inspect, proof, simulate, and validate how you collect data or serve experiences in your mobile app. This extension requires [`MobileCore`](https://github.com/adobe/aepsdk-core-android). 
 
-## About this project
+## Requirements
+
+- Android API 19 or newer
+
+## Installation
+
+Installation via [Maven](https://maven.apache.org/) & [Gradle](https://gradle.org/) is the easiest and recommended way to get the AEP SDK into your Android app. In your `build.gradle` file, include the latest version of following dependency:
+
+```
+implementation 'com.adobe.marketing.mobile:assurance:2.x.x'
+```
+
+## Development
+
+### Setup
+
+1. Fork this repo for your username and clone it on your development machine.
+2. If you haven't already, install Android Studio on your development machine. This project has been verified to work with Andrid Studio Dolphin.
+3. Open `aepsdk-assurance-android/code/settings.gradle` file in AndroidStudio
+
+You should now be able to see `assurance` and `assurance-testapp` modules under the `Project --> Android` section.
+
+### Build
+
+Navigate to the project root directory via command line and run the command:
+-  `make ci-build` to build the `assurance` library 
```

Comment:
nit: Can we remove the `ci-` prefix for these targets as we do nothing specific for CI?
We can keep the prefix only for relavent targets.

Reply:
Done. Changed the target names to remove ci- prefix for those which are not ci related.